### DISCUSSION TITLE
Remove old achievements

### DISCUSSION THREAD

**Discussion by mitchellvanbijleveld** [community/community#35259] [Other Feature Feedback, Questions, & Ideas]
I got an achievement with a repo that doesn't exist anymore. I don't like the fact that this achievement is untraceable and I want to get it removed from my profile because I basically can't verify I deserved this. It's more like a ghost achievement right now. I'd like to regain the achievement once I complete the requirements to get it. The 'inaccessible' below the achievement makes no sense to me.
<img width="439" alt="Schermafbeelding 2022-10-05 om 23 43 12" src="https://user-images.githubusercontent.com/106330077/194169687-e9fe92b1-f2ee-4ed8-a0f0-7054a45baedd.png">

**Comment by radar-blip** [comment]
But you did do something to achieve it, it wouldn't show up if you didn't.
The 'inaccessbile' just means the repo has been deleted/removed. It's not a big deal.

Other than that, you could always disable your achievements from showing (if you're really concerned).

**Reply by t1m013y** [reply]
Also 'inaccessible' may mean private repository.

**Comment by 0einstein0** [comment]
Wow

**Comment by jfullstackdev** [comment]
achievements, once received, are tied to your email, like the Artic badge, other than you posted it here and maybe Staff members will see this and help you, there is the Support, if you are really into it, but again, there is no assurance

https://support.github.com/contact/account

**Comment by mwong3la** [comment]
Considering the fact its an achievement.I dont think its a good thing wanting it removed.Let it stay.

**Reply by PurpleMonkey729** [reply]
I am sympathetic.
Achievements received in the past are your efforts, so it is better to leave them as it is.

**Comment by Anmol-Baranwal** [comment]
I understand your issue. Let's take a simple scenario:

I want to test Pair Extraordinaire in my GitHub private repository, and how it actually works. But it actually counts as an achievement and thus I deleted the repo (which most of them do) since the contribution to the private repository doesn't make it worth it. But what happens is, it becomes inaccessible. 
I think this issue should be addressed and all the inaccessible should automatically be adjusted to the conditions.

If the repository or the contribution doesn't exist, why should it be included right !!

**Reply by Anmol-Baranwal** [reply]
@mitchellvanbijleveld 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
